Tools/DataStorage/DataStorage.cs

```diff
@@ -10,6 +10,169 @@ namespace BD_oneLove.Tools.DataStorage
 {
     internal class DataStorage : IDataStorage
     {
+
+        public bool AddPlan(Plan p)
+        {
+            string sql = $"INSERT INTO [plan] (st_year, date_term1, date_term2, date_year) VALUES ('{p.StYear}', " +
+                        $"'{p.DateTerm1}', '{p.DateTerm2}', '{p.DateYear}'); ";
```

Comment:
Change this to parameters, we don't know the way environment on user computers manage data strings by default.